ibm_was/tests/test_e2e.py

```diff
@@ -0,0 +1,22 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.base import AgentCheck
+
+from . import common
+
+pytestmark = pytest.mark.e2e
```

Comment:
Do we prefer this to the @ notation?

Reply:
Both are ok, but since the whole file is e2e `ibm_was/tests/test_e2e.py`, that avoids repeating the `pytest.mark.e2e` if we have multiple tests.